test/common/router/config_impl_test.cc

```diff
@@ -2466,7 +2466,7 @@ TEST_F(RouteMatcherTest, GrpcRetry) {
                 .retryOn());
 
   EXPECT_EQ(std::chrono::milliseconds(0),
-            config.route(genHeaders("www.lyft.com", "/foo", "GET"), 0)
+            config.route(genHeaders("www.lyft.com", "/bar", "GET"), 0)
```

Comment:
This seems to be another copy/paste error similar to the one I fixed yesterday, L2453 was checking the same thing